src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala

```diff
@@ -53,49 +51,55 @@ object MissingLinkPlugin extends AutoPlugin {
   override def requires: Plugins = JvmPlugin
   override def trigger: PluginTrigger = allRequirements
 
+  //missinglink-core has non-threadsafe caches
+  val missinglinkConflictsTag = Tags.Tag("missinglinkConflicts")
   val configSettings: Seq[Setting[_]] = Def.settings(
```

Comment:
And here, could you ass a blank line between those two lines. (We also have a blank line between two class members as soon as one of them spans several lines (which is the case of `configSettings`).